### DISCUSSION TITLE
Sanctum Question about strange line of code

### DISCUSSION THREAD

**Discussion by myckhel** [laravel/framework#36419] [Q&A] [answered]
I have a question about this line.
https://github.com/laravel/sanctum/blob/eb191ddfc3ec04bbead33593bf982e871095f25c/src/Guard.php#L55

Why is the code getting config's sanctum.guard when there were never a guard in the sanctum's config?

And i dont think laravel documented anything about sanctum's guard config.

Does that mean that we can change the guard from web to something else?

Well i tried putting `guard => 'api'` but my app responded with err connection reset error.

**Comment by golonix** [comment, marked as answer]
> Does that mean that we can change the guard from web to something else?

Exactly, you can set a different guard (as long as it is defined in `auth.php` config file, under `guards` key). What's more, you can set multiple guards to check (via array). 

> And i dont think laravel documented anything about sanctum's guard config.

I believe you're right, documentation says nothing about it. Surely, you can contribute to it and document such functionality 😄
